backend/backend.go

```diff
@@ -99,6 +103,10 @@ type Operation struct {
 	// Input/output/control options.
 	UIIn  terraform.UIInput
 	UIOut terraform.UIOutput
+
+	// If LockState is true, the Operation must Lock any
+	// state.Lockers for its duration, and Unlock when complete.
```

Comment:
Add to the comment: if using `backend.Local`, it is up to the caller to unlock the state.